qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestBase.java

```diff
@@ -103,20 +92,11 @@ public void failed(Throwable t, Description test) {
 
 	public TestBase() {
 		TestProperties testProperties = TestProperties.instance();
```

Comment:
do you still need this instance?

Reply:
ohh yes we need this initialization of testProperties, we only getting rid of saurce labs.

Reply:
where is it used?

Reply:
@sherrif10 it would be better for @Muta-Jonathan to respond to reviews since he is the PR author than you responding to them on some one else's PR.

Reply:
i agree with you.

Reply:
Thanks @kdaud @sherrif10 ...... have updated the PR